src/app/modules/map-config/components/dialog-palette-selector/dialog-palette-selector.component.html

```diff
@@ -1,10 +1,10 @@
 <div *ngIf="!this.selectedPalette">
     <h1 mat-dialog-title>{{ 'Select a template palette' | translate}}</h1>
```

Comment:
I suggest to rename the title to : Select a colour palette
Change the french translation to : "Sélectionner une palette de couleurs."